openml/runs/run.py

```diff
@@ -114,7 +115,14 @@ def from_filesystem(cls, folder):
             with open(trace_path, 'r') as fp:
                 trace = arff.load(fp)
                 run.trace_attributes = trace['attributes']
-                run.trace_content = trace['data']
+                data_tmp = trace['data']
+                # TODO move this functionality to trace.py and serialize/deserialize the trace
```

Comment:
Yes, I currently don't have the time. This is related to #482 and should be resolved in there.

Reply:
I created issue #483. The casts are necessary because arff only defines values to be numerical, not float or int. The cast to float isn't there any more, right?

Reply:
I assume not, as all other columns should be float as well (the predictions per class and the 'predicted' and 'correct' class)

The thing I am mostly worried about is the 'assumptions' in the code, i.e., if we ever add one column we don't have the overview where these kind of manual patches are. Therefore I would propose to move this part to the trace file anyway (I can do this later today, if that helps)

Reply:
They are all represented as float, regardless whether they are floats or ints. I totally agree, if you have the time to implement #483 this would definitively be great.

Reply:
No problem. In that case I guess you would have to also review this PR.

Reply:
I moved the functionality to the trace.py. While doing so, I realized that the current way of representing the trace (in arff) is quite hacky, and we should use the actual trace object throughout the run. This update will make things way more neat, but it will cost several hours to implement, which I can't do / test right now. 

I think the current solution moves the functionality to exactly where one would expect it to be and is fine for now. Furthermore, I made it a private function, so we don't have a real obligation to maintain it in future versions.